### DISCUSSION TITLE
Dependency versions in projects

### DISCUSSION THREAD

**Discussion by Timongcraft** [modrinth/.github#51] [Feature requests]
At the moment you can add dependencies to your project with different types like required, embedded etc. but it lacks the feature of choosing the version because of that people maybe download a incompatible version. And for embedded dependencies if the dependency gets a new version that is displayed there which is wrong.

**Comment by Minenash** [comment]
You can though?
![Screenshot_20230511-094047(1)](https://github.com/modrinth/.github/assets/12193049/6eb7796f-51c7-4ec8-99ef-0c96f1e2ca02)

**Reply by Timongcraft** [reply]
Oh yea, I thought that was to add version of the same project.
Thanks I'm dumb
